cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -180,6 +191,12 @@ module ice_grid
       logical (kind=log_kind), private :: &
          l_readCenter ! If anglet exist in grid file read it otherwise calculate it
 
+      interface grid_average_X2Y
+         module procedure grid_average_X2Y_1C , &
+                          grid_average_X2Y_1fC, &
+                          grid_average_X2Y_2C
```

Comment:
Are there more meaningful suffixes that could be used for these subroutine names?  Or are these already meaningful but not intuitive?

Reply:
These are purely internal as part of the overloaded interface and evolved organically, but I'll improve.

Reply:
Ah, so C = Carbon

Reply:
updated.